<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartialGameWindow</class>
 <widget class="QDialog" name="PartialGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pin2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>0</y>
     <width>53</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin8">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>200</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>450</y>
     <width>211</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Súčet  kolkov</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin6">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>200</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin1">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>400</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>191</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Počet hodov</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="roundsLCD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="scoreLCD">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>490</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pin3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>310</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pointsLCD">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stavButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Upraviť stav</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stavanieButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stavanie</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="koniecButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Koniec hry </string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>191</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Zhodene</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>191</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>DORAZKA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ninepins.qrc"/>
 </resources>
 <connections/>
</ui>
